CHANGELOG.rst

```diff
@@ -3,7 +3,7 @@
 
 **Changes**
 
-*
+* fix (`#1618`_): properly consider explicitly blocked plugins
```

Comment:
I fear this is too "pytest-internal-specific" and users won't really know what's going on... suggestion:

> Fix #1618: consider `--plugins` (`-p`) option before loading installed plugins, so it can be used to disable
  plugins with problems or incorrectly installed (by using `-p no:<plugin name>`).